src/scheduler/sequence.rs

```diff
@@ -168,17 +168,22 @@ impl _Sequence {
 
     fn append_token_to_blocks(&mut self, token: usize) {
```

Comment:
Updated it here to match mistral.rs's implementation - it's a bit more clear I think.

Reply:
match last {
            Some(last) => {
                last.append_token_id(tok);
            }

Do we need to check if last is full when adding some tokens into it?

Reply:
No, because we always check afterward if we filled up the block and then append one.

Reply:
> No, because we always check afterward if we filled up the block and then append one.

That's good.